packages/core/admin/admin/src/pages/Home/components/Widget.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+
+import { Box, Flex, Loader, Typography } from '@strapi/design-system';
+import { PuzzlePiece } from '@strapi/icons';
+import { type MessageDescriptor, useIntl } from 'react-intl';
+
+interface WidgetProps {
+  title: MessageDescriptor;
+  icon?: typeof import('@strapi/icons').PuzzlePiece;
+  children: React.ReactNode;
+  isLoading?: boolean;
+}
+
+const Widget = ({ title, icon = PuzzlePiece, isLoading = false, children }: WidgetProps) => {
+  const { formatMessage } = useIntl();
+  const Icon = icon;
+
+  return (
+    <Flex
+      direction="column"
+      alignItems="flex-start"
+      gap={4}
+      background="neutral0"
+      padding={6}
+      shadow="tableShadow"
+      hasRadius
+    >
+      <Flex direction="row" alignItems="center" gap={2}>
+        <Icon fill="neutral500" />
+        <Typography textColor="neutral500" variant="sigma">
+          {formatMessage(title)}
+        </Typography>
+      </Flex>
+      <Box width="100%" height="256px" overflow="auto">
+        {isLoading ? (
```

Comment:
what about error? 🤔

Reply:
I'm not sure if the error state will be common between all widgets or not. We don't have error state designs so I just asked @lucasboilly for them

Reply:
Just a thought, couldn't we have children here and create a similar API to [Page Helpers](https://github.com/strapi/strapi/blob/775e90d48ce6fe0aa3fbb16e78059d805e15178d/packages/core/admin/admin/src/components/PageHelpers.tsx#L1)? Like Widget.Loading, Widget.Empty, Widget.Error? I wonder if we could just use the page ones actually 🤔